process/transaction/shardProcess.go

```diff
@@ -1380,6 +1429,43 @@ func (txProc *txProcessor) executeFailedRelayedUserTx(
 	return nil
 }
 
+func (txProc *txProcessor) executeFailedRelayedV3UserTx(
```

Comment:
I don't think we need to execute any failed relayv3 usertx, as there is only one tx, where the fee is paid by a relayer.

Reply:
updated